ocs_ci/ocs/constants.py

```diff
@@ -2202,3 +2203,24 @@
     "rule3": UI_INPUT_RULES_GENERAL["rule2"],
     "rule4": UI_INPUT_RULES_GENERAL["rule3"],
 }
+
+# Stretch cluster
+STRETCH_CLUSTER_NAMESPACE = "sc-project"
+LOGREADER_JOB_PATH = os.path.join(TEMPLATE_STRETCH_CLUSTER_DIR, "logreader.cephfs.yaml")
```

Comment:
We already have the ceph logreader and writer yamls [here](https://github.com/red-hat-storage/ocs-ci/tree/master/ocs_ci/templates/workloads/logwriter) . Is this logreader something different than the existing one?